CODEOWNERS

```diff
@@ -1 +1 @@
-* @dakanev-msft @davidahs @AndreyTretyak @neilr81 @K-Cully @doguva-msft @aszczepanski @vlad-ion @armhil @muiriswoulfe @sebbe33 @artempushkin
+* @dakanev-msft @davidahs @AndreyTretyak @neilr81 @K-Cully @doguva-msft @aszczepanski @vlad-ion @armhil @muiriswoulfe @sebbe33 @artempushkin @gikoning
```

Comment:
You could remove doguva-msft from this list

Reply:
I think we could do it in a separate PR, I'll complete this one